### PR TITLE
Update Troubleshooting.

### PR DESCRIPTION
**Pull request by kattni**
Clarify things, update URLs. Add in the `storage.erase_filesystem()` method, and remove errant `.` that confused a user.

### CODE REVIEW

File: docs/troubleshooting.rst [hunk 1]
```diff
@@ -19,24 +19,36 @@ happen on Linux and Mac OSX but it's less likely.
...
+Erase File Method
+^^^^^^^^^^^^^^^^^
+**If you do not have access to the REPL, you have the following options to erase your board.**
+
 **For boards with** ``CIRCUITPY`` **stored on a separate SPI flash chip,
 such as Feather M0 Express, Metro M0 Express and Circuit Playground Express:**
 
-
-#. Download the appropriate flash .erase uf2 from `the Adafruit_SPIFlash repo <https://github.com/adafruit/Adafruit_SPIFlash/tree/master/examples/flash_erase_express>`_.
+#. Download the appropriate flash erase .uf2 from `the Adafruit_SPIFlash repo <https://github.com/adafruit/Adafruit_SPIFlash/tree/master/examples/flash_erase_express>`_.
```
Comment (dhalbert) [inline docs/troubleshooting.rst:38]: There aren't any .uf2's in the release directory in this repo. Maybe we made them by hand a long time ago. The alternative is to use Arduino and upload the program, or use the precompiled ones that are in the guide.

**Review by dhalbert — CHANGES_REQUESTED** [review]

File: docs/troubleshooting.rst (hunk 1 above)
Comment (kattni) [reply docs/troubleshooting.rst:38]: @dhalbert Should I simply rewrite both of these sections to point to the CIRCUITPY Drive Issues section on the Troubleshooting page in the Welcome to CircuitPython guide? It's clearly explained there, and I don't think duplicating all of that here makes sense.

Comment (kattni) [reply docs/troubleshooting.rst]: Updated to point to guide. Added section to Troubleshooting page on getting into safe mode, and linked to that as well.

**Review by dhalbert — APPROVED** [review]
Thanks!

**Pull request merged**